operatorcourier/cli.py

```diff
@@ -123,6 +123,9 @@ def parse(self):
             'extracted bundle files')
         flatten_parser.set_defaults(func=self.flatten)
 
+        if len(sys.argv) == 1:
```

Comment:
this check should be added after `parser.parse_args()` because `operator-courier --version` is valid

Also I'd go with checking presence of `arg.func` instead testing number of args.

Reply:
Thanks. I've added a check for the presence of `func` and if it's callable.